src/index.html

```diff
@@ -6,7 +6,71 @@
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="page">
+    <div class="container">
+      <div class="calendar calendar--month-length--29">
+        <div class="calendar__day calendar__day--start--sun"></div>
```

Comment:
it is not necessary to add this class `calendar__day--start--sun` you can use `:first-child()` in css